package.json

```diff
@@ -1,29 +1,29 @@
 {
-  "name": "Gibberish-AES",
-  "version": "1.0.0",
-  "author": "Mark Percival <mark@mpercival.com>",
-  "description": "A lightweight Javascript Libray for OpenSSL compatible AES CBC encryption.",
-  "contributors": [
-    {
-      "name": "Mark Percival",
-      "email": "mark@mpercival.com"
+	"name": "Gibberish-AES",
+	"version": "1.0.0",
+	"author": "Mark Percival <mark@mpercival.com>",
+	"description": "A lightweight Javascript Libray for OpenSSL compatible AES CBC encryption.",
+	"contributors": [
+		{
+			"name": "Mark Percival",
+			"email": "mark@mpercival.com"
     }
   ],
-  "main": null,
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/mdp/gibberish-aes.git"
-  },
-  "keywords": [],
-  "dependencies": {},
-  "devDependencies": {
-    "grunt": "0.4.x",
-    "grunt-contrib-uglify": "0.x.x",
-    "grunt-contrib-clean": "0.x.x",
-    "grunt-contrib-copy": "0.x.x",
-    "grunt-contrib-jshint": "0.x.x"
-  },
-  
-  "bundledDependencies": [],
-  "license": "MIT"
-}
+	"main": "dist/gibberish-aes-1.0.0.js",
```

Comment:
This will break as soon as that version number goes up. May as well do the same thing as the `component.json` file and point to `src/gibberish-aes.js` instead. All the grunt task does is copy the source file and applies the version suffix to the file name anyway.